k2/csrc/ragged.h

```diff
@@ -72,6 +72,10 @@ class RaggedShape {
     }
   }
 
+  // Append `other` to `*this`.  `other` must have the same number of axes as
+  // `this`.  The reason this is
+  void _Append(const RaggedShape &other);
```

Comment:
why is there a `_` here? seems typo?

Reply:
I'll remove it.  Was going for a Pytorch-style convention but it's not consistent right now.